captura/tests.py

```diff
@@ -840,3 +840,60 @@ def test_upload_photo_bad_request(self):
                       kwargs={'id_estudio': self.estudio1.pk})
         response = self.client.get(url)
         self.assertEqual(400, response.status_code)
+
+    def test_delete_modal_photo(self):
+        """ This test checks that the view 'captura:upload_photo', allows
+        the upload of a new family photo.
+        """
+        url = reverse('captura:upload_photo',
+                      kwargs={'id_estudio': self.estudio1.pk})
+        test_image = settings.BASE_DIR + static('test_files/borrosa.jpeg')
+        with open(test_image, 'r+b') as testing:
+            form = {'estudio': self.estudio1.pk,
+                    'file_name': 'prueba',
+                    'upload': testing}
+            response = self.client.post(url, form)
+            self.assertEqual(302, response.status_code)
+            image = Foto.objects.filter(estudio=self.estudio1).last()
+
+            # Check that the object and file exist
+            response = self.client.get(reverse('captura:form_delete_foto',
+                                               kwargs={'id_foto': image.pk}),
+                                       HTTP_X_REQUESTED_WITH='XMLHttpRequest')
+            self.assertEqual(200, response.status_code)
+            self.assertTrue(b'seguro que desea borrar esta foto' in response.content)
+
+            response = self.client.post(reverse('captura:delete_foto',
+                                                kwargs={'id_foto': image.pk}),
+                                        {'id_foto': image.pk})
+            self.assertRedirects(response, reverse('captura:list_photos',
+                                                   kwargs={'id_estudio': image.estudio.pk}))
+
+    def test_upload_and_delete_photo(self):
+        """ This test checks that the view 'captura:upload_photo', allows
```

Comment:
Same with this test, check the description, as it does not reflect the purpose of the test.